.github/workflows/assign-author.yml

```diff
@@ -0,0 +1,13 @@
+name: Assign Author
+
+on:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  assignAuthor:
+    name: Assign author to PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Assign author to PR
+        uses: technote-space/assign-author@v1
```

Comment:
This seems out of scope, why is this necessary? I want to keep the amount of untrusted third-party action (basically not from GitHub/Microsoft/etc.) limited. It's a bad idea to run arbitrary actions because this is a critical supply chain asset.

Reply:
Removed